### PR TITLE
Cover SUSE's systemd patches in SLE 15

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
## Description

- Add it also for SLE-15

## Ticket

- [poo#23798](https://progress.opensuse.org/issues/23798)

## Verification run

- [copland#1552](http://copland.arch.suse.de/tests/1552)

### CODE REVIEW

File: products/sle/main.pm [hunk 1]
```diff
@@ -1392,6 +1392,10 @@ elsif (get_var('HPC')) {
         }
     }
 }
+elsif (get_var('SYSTEMD_TESTSUITE')) {
+    boot_hdd_image;
+    loadtest 'console/systemd_testsuite';
+}
```
Comment (okurz) [inline products/sle/main.pm]: [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html). How about in lib/main_common.pm?

**Comment by SeroSun** [comment]
I don't understand, why we need this test? Since systemd test already covered in userspace job group. https://openqa.suse.de/tests/1196422#step/userspace_systemd/1
It fails because of poo#25846, jtzhao is fixing it.

**Comment by SergioAtSUSE** [comment]
Hi @SeroSun, the test that I am adding is not to test systemd itself, but to test the patches from SUSE that are not accepted on upstream for being too specific or for some other reason.
Please notice that the test suite used in this test is not the upstream one, but the modified version from systemd developers to test their patches. And the patches are not automatically tested anywhere.

File: products/sle/main.pm (hunk 1 above)
Comment (SergioAtSUSE) [reply products/sle/main.pm]: I don't understand what you mean. Which code am I repeating?

**Comment by SeroSun** [comment]
Hi Sergio, good to know. Thanks.

**Comment by SergioAtSUSE** [comment]
New verification run: http://copland.arch.suse.de/tests/1565#step/systemd_testsuite/26

The test fails because of missing packages on SLE.

**Pull request merged**